src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/ComputeWasmBuildAssets.cs

```diff
@@ -83,10 +83,9 @@ public override bool Execute()
                 return true;
             }
 
-            if (AssetsComputingHelper.TryGetAssetFilename(CustomIcuCandidate, out string customIcuCandidateFilename))
+            if (!AssetsComputingHelper.TryGetAssetFilename(CustomIcuCandidate, out string customIcuCandidateFilename))
```

Comment:
`CustomIcuCandidate` is not required so we should add to the check: CustomIcuCandidate != null

Reply:
If we succeed then we proceed to for loop where `customIcuCandidateFilename` is processed. In the original code we were adding custom file to `assetCandidates` twice - once in this if and then in the loop.

Reply:
That's the error we produce when we keep adding it twice:
```
artifacts/bin/dotnet-latest/sdk/8.0.100-rc.1.23410.12/Sdks/Microsoft.NET.Sdk.StaticWebAssets/targets/Microsoft.NET.Sdk.StaticWebAssets.targets(475,5): error : Conflicting assets with the same target path '_framework/icudt.dat'. 
```